nebula1/visor_run_config.toml

```diff
@@ -0,0 +1,23 @@
+name = "<insert_vega_version_here>"
+
+[vega]
+  [vega.binary]
+    path = "vega"
+    args = [
+      "start",
+      "--home", ".../vega_home",
+      "--tendermint-home", ".../cometbft_home",
+      "--nodewallet-passphrase-file", ".../.passphrase.txt",
```

Comment:
should we add the `--network-url=https://raw.githubusercontent.com/Nebula-DEX/networks/main/nebula1/genesis.json` flag here?

Reply:
Let me test it quickly